core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -264,6 +270,14 @@ var TopicEditorPage = function() {
     var pageEditorInput = pageEditor.element(by.css('.oppia-rte'));
     await pageEditorInput.click();
     await pageEditorInput.sendKeys(htmlContent);
+    var conceptCardButton = pageEditor.element(by.css('#cke_24'));
+    await conceptCardButton.click();
```

Comment:
Can you use `action.click` here instead?

Reply:
Done